Update readme to new chart interface

pluehne opened the pull request:
The HTML/JSON interface for creating charts was recently changed. This updates the readme accordingly. Additionally, this adds missing documentation about the allowed configuration options.

pluehne commented:
@filmaj: As you suggested, I updated the readme concerning charts to the new format that I introduced earlier.

Unfortunately, I just noticed that the configuration options part might interfere with your issue #82, where you said that you worked on something similar. I didn’t put too much effort into the changes above, so if you have a better idea how to describe the options, let’s take yours and just keep my description of the HTML structure 😃.

filmaj commented:
@pluehne I haven't started on those so I think we should what you have here.

filmaj commented on docs/README.md:
```diff
@@ -5,14 +5,58 @@ This web application is built as a [Jekyll](https://jekyllrb.com/) site and is i
 
 Each `.html` page in this directory contains one or more charts.
-Each chart is defined as an `<svg>` element with a few special attributes that define their behavior:
+Charts are created from `<div>` containers of the class `chart-placeholder` by [`charts.js`](assets/js/charts.js).
```
Perhaps we can word this as "Charts are dynamically created by [`charts.js`](assets/js/charts.js) from `<div class="chart-placeholder">` placeholders present in the HTML" ?

pluehne replied:
Thanks, I’ll incorporate your feedback!

filmaj commented on docs/README.md:
```diff
@@ -5,14 +5,58 @@ This web application is built as a [Jekyll](https://jekyllrb.com/) site and is i
+    - `history`: time series charts for tracking metrics over time
+    - `list`: horizontal bar charts for general numeric data
+    - `table`: tables directly rendered from the TSV source
+    - `collaboration`: chord diagrams used to visualize connections between entities
```
While Hubble's sole chord diagram today is the Collaboration chart, perhaps it would align better to rename this chart type to 'chord'? This way if in the future we ship a _different_ kind of chord diagram, the type would apply a bit more generically?

pluehne replied:
Good point, I’ll change the name to `chord`!

pluehne commented:
@filmaj: I’ve addressed your two comments. I rebased your phrasing suggestion into the original commit, and I added a second commit for renaming the collaboration chart to *chord chart*.

filmaj commented:
Looks great!
